api/napi/index.ts

```diff
@@ -144,4 +145,75 @@ export class ArrayModel<T> implements Model<T> {
     entries(): IterableIterator<[number, T]> {
         return this.a.entries()
     }
+}
+
+export class Component {
```

Comment:
In light of the public Rust interpreter API, perhaps this should be called `ComponentInstance`.

In neon that was private API, but in this change it becomes public, so the name becomes visible.

Reply:
@tronical the Rust variant of this is called `ComponentHandle` https://docs.rs/slint/1.2.2/slint/trait.ComponentHandle.html, should we name in in JS/TS also `ComponentHandle`? We have already a `ComponentInstance` in the interpreter.

Reply:
Yeah, I guess `ComponentHandle` works, too, if the type is changed to be an interface and the factory function returns that, and this wrapper becomes private API then. But `Component` itself is wrong, I think, since it's not a `Component` but an instance of it.